### PR TITLE
Added gems simplecov and awesome print

### PR DESCRIPTION
**Pull request by EGimenoS**
Just added two little but somehow useful gems.

Simplecov: Measures the percentage of our code which is covered by tests. When you run bundle exec rpsec, you see the result of the analysis as a %, then you can check this folder for a graphical report:
/vagrant/src/team-chess-app/coverage

Awesome_Print: I like it to make easier working in the console, specially when querying the database. Just have to type "ap" before the query to obtain a "prettified" version of the result.

for example irb> ap Game.all

### CODE REVIEW

**Comment by EGimenoS** [comment]
I think I have it, placed coverage/ into gitignore and typed git rm -r --cached coverage to delete the folder from the tracked ones. I placed both gems into their respective groups.

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,6 @@
 /log/*
 !/log/.keep
 /tmp
+
+coverage/
```
Comment (lasley) [inline .gitignore]: `coverage/*`

**Comment by saakuffo** [comment]
Ernesto, are you going to do the online hooks?

**Comment by lasley** [comment]
(I recommend CodeCov)

**Comment by EGimenoS** [comment]
sure, I will check how can be done the integration between codecov and codeship.

**Comment by saakuffo** [comment]
reserve conflict and I will approve and merge.
We can have the integration be another PR. Smaller commits :)

**Comment by lasley** [comment]
There are no code changes required for the integration to CI.

**Comment by lasley** [comment]
Any word on this?

**Comment by EGimenoS** [comment]
I haven't been able to sync the team repo with codecov https://codecov.io/gh, don't know if not being admin is the reason.  Code regarding I think it's ok as this branch has simplecov configured.

**Comment by lasley** [comment]
@EGimenoS - I just gave you admin. You will need to follow the instructions from the bottom of Codecov:
```
Still nothing? Check your Application Settings in GitHub. Click "Grant Access" in your team list.
```

**Comment by EGimenoS** [comment]
I think it's done, now the repo is associated to codecov and I can see how codecov reports one commit with 99.65% coverage.

**Comment by lasley** [comment]
Will need to resolve conflicted lock btw

**Comment by lasley** [comment]
👍

**Comment by EGimenoS** [comment]
Please let me know if we can merge

**Comment by lasley** [comment]
@EGimenoS - As always, two approvals are required for merge unless critical & obvious breakfix

**Pull request merged**